src/app/views/query-runner/request/RequestHeaders.tsx

```diff
@@ -0,0 +1,63 @@
+import { IconButton, TextField } from 'office-ui-fabric-react';
+import React from 'react';
+
+interface IRequestHeadersControl {
+    handleOnHeaderDelete: Function;
+    handleOnHeaderNameChange: Function;
+    handleOnHeaderValueChange: Function;
+    handleOnHeaderValueBlur: Function;
+    headers: Array<{ name: string; value: string; }>;
+}
+
+export const RequestHeadersControl = ({
+    handleOnHeaderDelete,
+    handleOnHeaderNameChange,
+    handleOnHeaderValueChange,
+    handleOnHeaderValueBlur,
+    headers,
+}: IRequestHeadersControl) => {
+    const headersList = (
+        headers.map((header, index) => {
+            return (
+                <tr key={index}>
+                    <td>
+                        <TextField
+                            className='header-input'
+                            onChange={(event, name) => handleOnHeaderNameChange(event, name)}
```

Comment:
The event argument is not being used, you can consider not passing it to handleOnHeaderNameChange